src/site/content/en/blog/module-workers/index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Threading the web with module workers
+subhead: |
+  Moving heavy lifting into background threads is now easier with JavaScript modules in web workers.
+date: 2019-12-17
+hero: hero.jpg
+alt: Computer processor graphic
+authors:
+  - developit
+description: |
+  Module workers make it easy to unblock the main thread by moving expensive code to a background
+  thread while keeping the ergonomic and performance benefits of standard JavaScript modules.
+tags:
+  - post
+  - web-workers
+  - js-modules
+---
+
+<!--lint disable no-heading-punctuation heading-increment -->
```

Comment:
```suggestion
```

Presumably this is instructing the linter to ignore the checks around headings levels. That's actually something that we must fix before publishing this.